src/plugins/imagery/components/ImageryView.vue

```diff
@@ -620,6 +634,8 @@ export default {
 
         this.listenTo(this.focusedImageWrapper, 'wheel', this.wheelZoom, this);
         this.loadVisibleLayers();
+        // // set after render so initial scroll event is skipped
+        setTimeout(this.setScrollBehavior, 3 * 1000);
```

Comment:
This seems like it may be prone to race conditions. Is there a way we can detect the initial scroll event instead?

Reply:
I'll have to look into that, I put this in temporarily since waiting for the initial render completion wasn't sufficiently long in time. I did try capturing this on the initial scroll event, but it fired before the data was present and so it still appeared to have the excessively long scroll on initial load.